react-backend/client/src/components/visitors/qrprivacy.js

```diff
@@ -1,30 +1,27 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Paragraph } from '../../shared/components/text/base';
 
-const qrPrivacy = props => (
-  <aside className={`Privacy ${props.className}`}>
-    <img className="Privacy__image" src="/images/question.png" alt="question" />
-    <h3 className="Privacy__title">What information are we storing?</h3>
-    <p className="Privacy__text">
+const ATag = styled.a`
+`;
+const qrPrivacy = () => (
+  <aside>
+    <h3>What information are we storing?</h3>
+    <Paragraph>
       Your name, sex, age, and reason for visiting today, as well as the time and date of your
       visit.
       <br />
       <br />
-    </p>
-    <h3 className="Privacy__title">Why are we collecting this information?</h3>
-    <p className="Privacy__text">
+    </Paragraph>
+    <h3>Why are we collecting this information?</h3>
+    <Paragraph>
       We are always trying to improve what we offer you. By signing in, you are helping us make
       our services better.
-    </p>
+    </Paragraph>
+    <ATag href="http://www.powertochange.org.uk/data-protection-funding-applications/">
+      Data Protection Policy
```

Comment:
There's already `Link` styled tag in the text base, this is not in keeping with the designs.

Reply:
This is an external link. I assumed you couldn't use `Link` from `react-router-dom` for external links.

Reply:
Ah, true. Fair enough, but according to the design, it still needs to look like the other links.

Reply:
I'll nab your styling from `Link`